Game/Source/GDKTestGyms/Private/TestGymsReplicationGraph.cpp

```diff
@@ -481,6 +560,45 @@ void UTestGymsReplicationGraphNode_AlwaysRelevant_ForConnection::ResetGameWorldS
 	AlwaysRelevantStreamingLevelsNeedingReplication.Empty();
 }
 
+void UTestGymsReplicationGraphNode_AlwaysRelevant_ForConnection::GatherClientInterestedActors(const FConnectionGatherActorListParameters& Params)
+{
+	InterestedActorList.Reset();
+	for (const FNetViewer& CurViewer : Params.Viewers)
+	{
+		InterestedActorList.ConditionalAdd(CurViewer.InViewer);
+		InterestedActorList.ConditionalAdd(CurViewer.ViewTarget);
+		if (APlayerController* PC = Cast<APlayerController>(CurViewer.InViewer))
+		{
+			if (APlayerState* PS = PC->PlayerState)
+			{
+				InterestedActorList.ConditionalAdd(PS);
+			}
+		}
+	}
+
+	Params.OutGatheredReplicationLists.AddReplicationActorList(InterestedActorList);
+
+	UTestGymsReplicationGraph* TestGymsGraph = CastChecked<UTestGymsReplicationGraph>(GetOuter());
+	TMap<FName, FActorRepListRefView>& AlwaysRelevantStreamingLevelActors = TestGymsGraph->AlwaysRelevantStreamingLevelActors;
+
+	for (int32 Idx = AlwaysRelevantStreamingLevelsNeedingReplication.Num() - 1; Idx >= 0; --Idx)
```

Comment:
Why backwards?

Reply:
Based on existing code in GatherActorListsForConnection.